code.js

```diff
@@ -64,9 +75,17 @@ const contestEndTime = Math.floor(Date.parse(endTime._i) / 1000);
             if (item.parentElement.className === "text-center no-break") continue;
 
             const hProblemId = hpath[hpath.length - 1];
-            changeProblemTitle(hProblemId, estimatedDifficulties, item, true);
+            if (hideDifficulty) {
+              problemListFunctions.push(() => changeProblemTitle(hProblemId, estimatedDifficulties, item, true));
+            }
+            else {
+              changeProblemTitle(hProblemId, estimatedDifficulties, item, true);
+            }
         }
-
+    }
+    if (hideDifficulty && problemListFunctions.length != 0) {
```

Comment:
ここでも[個別の問題ページ](https://atcoder.jp/contests/abc276/tasks/abc276_a)にネタバレ防止ボタンを挿入しようとしています。
結果としては`querySelector`に失敗してエラーが出ます。

すでに上でボタンを追加したので、ここではボタンを挿入しないようにしましょう。
条件に「個別の問題ページではないこと」を加えましょう。

```suggestion
        if (
      hideDifficulty &&
      problemListFunctions.length != 0 &&
      path[path.length - 2] !== "tasks"
    ) {
```

Reply:
こちら変更をコミットさせていただきました！
このifの前にインデントが有るのは意図的でしょうか？

Reply:
ミスです。
とりあえずPRは通して、
後でリポジトリにフォーマッターを導入して解決します。